components/odh-notebook-controller/controllers/notebook_runtime.go

```diff
@@ -105,7 +105,9 @@ func (r *OpenshiftNotebookReconciler) syncRuntimeImagesConfigMap(ctx context.Con
 				// Construct the key name
 				if displayName != "" {
 					formattedName := formatKeyName(displayName)
-					data[formattedName] = metadataParsed
+					if formattedName != "" {
+						data[formattedName] = metadataParsed
```

Comment:
@ada333 why didn't you remind me this is about naming configmap keys, and not configmaps themselves? I was talking completely offtopic yesterday ;(